src/components/views/Navbar/Navbar.tsx

```diff
@@ -2,7 +2,7 @@ import styles from "./Navbar.module.scss";
 import MobileNavigation from "./MobileNavigation";
 import Navigation from "./Navigation";
 
-const Navbar = (props) => {
+const Navbar: React.FC<{ isDesktop: boolean | null }> = (props) => {
   const isDesktop = props.isDesktop;
```

Comment:
I've noted this pattern quite often, where you basically add props de-structuring in the component body. It is fine. I know people who kinda hate de-structuring in the function parameters. I am not one of them, and don't think @pantiushenkov is as well. I honestly prefer not having undestructured props at all. It is not a big deal, just a stylistic _(-[ish](https://gist.github.com/Malien/f5eddf7b8c65935cdbd2f03f242ffdd1))_ preference